bin/os_main.tf

```diff
@@ -0,0 +1,103 @@
+#Openstack credentials and instance settings, SSH Variables and current image version
+variable username {}
+
```

Comment:
So, apparently the command `terraform fmt` is adding that space. If I remove it, then the git diff test in travis after the before mentioned command will fail.